<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>729</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Swarm Sailor</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item row="2" column="2">
     <widget class="QPushButton" name="Button_Open_Port">
      <property name="text">
       <string>Open Port</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" alignment="Qt::AlignTop">
     <widget class="QLabel" name="label_GNSS_status">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>GNSS:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_Messages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Messages</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="3">
     <widget class="QWidget" name="advancedSection" native="true">
      <layout class="QGridLayout" name="advancedSection_layout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="6" column="1">
        <widget class="QPushButton" name="Button_Serial_Monitor_Send">
         <property name="toolTip">
          <string>Send to Serial</string>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" rowspan="2">
        <widget class="QPlainTextEdit" name="Serial_Monitor_Display"/>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="Serial_Monitor_SendLine"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Label_Serial_Monitor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Serial Monitor</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line_3">
         <property name="toolTip">
          <string>Don't Spook the Meatloaf</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="Button_DeviceID">
           <property name="toolTip">
            <string>Get Device ID</string>
           </property>
           <property name="text">
            <string>DeviceID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Button_Firmware">
           <property name="toolTip">
            <string>Get Firmware</string>
           </property>
           <property name="text">
            <string>Check Firmware</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Button_Geospatial">
           <property name="toolTip">
            <string>Toggle GNSS data</string>
           </property>
           <property name="text">
            <string>Geospatial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Button_Restart">
           <property name="toolTip">
            <string>Restart Modem</string>
           </property>
           <property name="text">
            <string>Restart</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Button_Empty_TX">
           <property name="text">
            <string>Empty TX</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Button_Serial_Terminal_Clear">
           <property name="text">
            <string>Terminal Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="0" rowspan="4" colspan="2">
     <widget class="QPlainTextEdit" name="Messages_Display">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_Commands">
      <property name="text">
       <string>Commands</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_COM_PORT">
      <property name="text">
       <string>COM Port:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_PORT"/>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="Button_Get_GRIB">
      <property name="toolTip">
       <string>Request a new GRIB file</string>
      </property>
      <property name="text">
       <string>Request GRIB</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="Button_Send_Message">
      <property name="text">
       <string>Send Message</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QPushButton" name="Button_Mailbox">
      <property name="text">
       <string>Check Mailbox</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="data_GNSS">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_COM_Status">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Button_Refresh_PORT">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2" alignment="Qt::AlignBottom">
     <widget class="QPushButton" name="Button_Advanced">
      <property name="toolTip">
       <string>Close or Open Advanced Window</string>
      </property>
      <property name="text">
       <string>Advanced</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Label_Title">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>SWARM SAILOR GUI</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_Portal_HTTP">
      <property name="text">
       <string>&lt;a href=&quot;http://portal.swarmsailor.space/&quot;&gt;Web Portal&lt;/a&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2" alignment="Qt::AlignTop">
     <widget class="QLabel" name="data_Status">
      <property name="text">
       <string>Disconnected</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button_Close_Port">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Close Port</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
